x-pack/platform/plugins/shared/fleet/server/services/epm/agent/agent.ts

```diff
@@ -79,7 +90,6 @@ function buildTemplateVariables(logger: Logger, variables: PackagePolicyConfigRe
     // support variables with . like key.patterns
     const keyParts = key.split('.');
     const lastKeyPart = keyParts.pop();
-    logger.debug(`Building agent template variables`);
```

Comment:
nit: should this logger be removed?

Reply:
when debugging I found that log pretty verbose and I am not sure it bring so much value